app/router.js

```diff
@@ -41,9 +41,7 @@ AppRouter.map(function() {
     });
   });
 
-  this.route('project', {
-    path: '/:slugged_route_slug/:project_slug'
-  }, function() {
+  this.route('project', function() {
```

Comment:
Is this just for testing, too? This breaks a lot of the way the application works otherwise.